packages/theme-basic/src/scss/components/TextInput.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../scss/core';
```

Comment:
We should also include this file in `front.scss`, so that final css build contains this file as well.

Reply:
done

Reply:
`@import ../core` do not work?